share/dbtypes.json.in

```diff
@@ -0,0 +1,1064 @@
+    [0, "BOOL", "bool", 0, "HDF5", "v1.2", 1],
+    [0, "BOOL", "bool", 0, "SQLite", "v1.3", 1],
+    [1, "INT", "int", 0, "SQLite", "v1.3", 1],
+    [2, "FLOAT", "float", 0, "SQLite", "v1.3", 1],
+    [3, "DOUBLE", "double", 0, "SQLite", "v1.3", 1],
+    [4, "STRING", "std::string", 1, "SQLite", "v1.3", 1],
+    [5, "VL_STRING", "std::string", 1, "SQLite", "v1.3", 1],
+    [6, "BLOB", "cyclus::Blob", 0, "SQLite", "v1.3", 1],
+    [7, "UUID", "boost::uuids::uuid", 0, "SQLite", "v1.3", 1],
+    [8, "VECTOR_BOOL", "std::vector<bool>", 1, "SQLite", "v1.3", 0],
+    [9, "VL_VECTOR_BOOL", "std::vector<bool>", 1, "SQLite", "v1.3", 0],
+    [10, "VECTOR_INT", "std::vector<int>", 1, "SQLite", "v1.3", 1],
+    [11, "VL_VECTOR_INT", "std::vector<int>", 1, "SQLite", "v1.3", 1],
+    [12, "VECTOR_FLOAT", "std::vector<float>", 1, "SQLite", "v1.3", 0],
+    [13, "VL_VECTOR_FLOAT", "std::vector<float>", 1, "SQLite", "v1.3", 0],
+    [14, "VECTOR_DOUBLE", "std::vector<double>", 1, "SQLite", "v1.3", 1],
+    [15, "VL_VECTOR_DOUBLE", "std::vector<double>", 1, "SQLite", "v1.3", 1],
+    [16, "VECTOR_STRING", "std::vector<std::string>", 2, "SQLite", "v1.3", 1],
+    [17, "VL_VECTOR_STRING", "std::vector<std::string>", 2, "SQLite", "v1.3", 1],
+    [18, "VECTOR_VL_STRING", "std::vector<std::string>", 2, "SQLite", "v1.3", 1],
+    [19, "VL_VECTOR_VL_STRING", "std::vector<std::string>", 2, "SQLite", "v1.3", 1],
+    [20, "VECTOR_BLOB", "std::vector<cyclus::Blob>", 1, "SQLite", "v1.3", 0],
+    [21, "VL_VECTOR_BLOB", "std::vector<cyclus::Blob>", 1, "SQLite", "v1.3", 0],
+    [22, "VECTOR_UUID", "std::vector<boost::uuids::uuid>", 1, "SQLite", "v1.3", 0],
+    [23, "VL_VECTOR_UUID", "std::vector<boost::uuids::uuid>", 1, "SQLite", "v1.3", 0],
+    [24, "SET_BOOL", "std::set<bool>", 1, "SQLite", "v1.3", 0],
+    [25, "VL_SET_BOOL", "std::set<bool>", 1, "SQLite", "v1.3", 0],
+    [26, "SET_INT", "std::set<int>", 1, "SQLite", "v1.3", 1],
+    [27, "VL_SET_INT", "std::set<int>", 1, "SQLite", "v1.3", 1],
+    [28, "SET_FLOAT", "std::set<float>", 1, "SQLite", "v1.3", 0],
+    [29, "VL_SET_FLOAT", "std::set<float>", 1, "SQLite", "v1.3", 0],
+    [30, "SET_DOUBLE", "std::set<double>", 1, "SQLite", "v1.3", 0],
+    [31, "VL_SET_DOUBLE", "std::set<double>", 1, "SQLite", "v1.3", 0],
+    [32, "SET_STRING", "std::set<std::string>", 2, "SQLite", "v1.3", 1],
+    [33, "VL_SET_STRING", "std::set<std::string>", 2, "SQLite", "v1.3", 1],
+    [34, "SET_VL_STRING", "std::set<std::string>", 2, "SQLite", "v1.3", 1],
+    [35, "VL_SET_VL_STRING", "std::set<std::string>", 2, "SQLite", "v1.3", 1],
+    [36, "SET_BLOB", "std::set<cyclus::Blob>", 1, "SQLite", "v1.3", 0],
+    [37, "VL_SET_BLOB", "std::set<cyclus::Blob>", 1, "SQLite", "v1.3", 0],
+    [38, "SET_UUID", "std::set<boost::uuids::uuid>", 1, "SQLite", "v1.3", 0],
+    [39, "VL_SET_UUID", "std::set<boost::uuids::uuid>", 1, "SQLite", "v1.3", 0],
+    [40, "LIST_BOOL", "std::list<bool>", 1, "SQLit
```

Comment:
Somewhere we should mention that this file should be updated every release.  Maybe that needs togo in the docs repo, though and shouldn't block this.